meta-ov/recipes-kernel/linux/files/openvario.cfg

```diff
@@ -25,7 +25,6 @@ CONFIG_UEVENT_HELPER=y
 CONFIG_INPUT_EVDEV=y
 # CONFIG_INPUT_FF_MEMLESS is not set
 # CONFIG_INPUT_MOUSE is not set
-CONFIG_INPUT_UINPUT=y
```

Comment:
I think touchscreen handling is based on uinput (`ts_uinput` daemon). If it is the same uinput, suggest to leave it enabled.

Reply:
It is still enabled; the line was duplicate, and this commit just removes one copy.